com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandardProgram.hlsl

```diff
@@ -27,6 +29,7 @@
 #pragma shader_feature_local_fragment _USE_SSAA
 #pragma shader_feature_local _ _DIRECTIONAL_LIGHT _DISTANT_LIGHT
 #pragma shader_feature_local _NON_PHOTOREALISTIC
+#pragma shader_feature_local _RECEIVESHADOW
```

Comment:
Can we change `_RECEIVESHADOW` to `_RECEIVE_SHADOWS_OFF` to match URP? See: https://github.com/Unity-Technologies/Graphics/blob/d84f56b68aea37785813fddfcbcf63384e9640ff/Packages/com.unity.render-pipelines.universal/ShaderLibrary/Shadows.hlsl#L10

Reply:
Sorry I'm late!
I was just cleaning up some New Year's work!
I'll check today/tomorrow !